iptables/match_builder.go

```diff
@@ -76,6 +76,20 @@ func (m MatchCriteria) MarkMatchesWithMask(mark, mask uint32) MatchCriteria {
 	return append(m, fmt.Sprintf("-m mark --mark %#x/%#x", mark, mask))
 }
 
+func (m MatchCriteria) MarkNotMatchesWithMask(mark, mask uint32) MatchCriteria {
```

Comment:
How about `NotMarkMatchesWithMask()`  (for consistency with the other `NotXXX` matches below)

Reply:
Done.